content/blog/run-your-own-rss-server/index.md

```diff
@@ -0,0 +1,258 @@
+---
+title: "Run Your Own RSS Server on AWS with Pulumi"
+
+# The date represents the post's publish date, and by default corresponds with
+# the date this file was generated. Posts with future dates are visible in development,
+# but excluded from production builds. Use the time and timezone-offset portions of
+# of this value to schedule posts for publishing later.
+date: 2019-09-13
+
+# Draft posts are visible in development, but excluded from production builds.
+# Set this property to `false` before submitting your post for review.
+draft: false
+
+# Use the optional meta_desc property to provide a brief summary (one or two sentences)
+# of the content of the post, which is useful for targeting search results or social-media
+# previews. If omitted or left blank, the content preceding the `<!--more-->` token
+# will be used in its place.
+meta_desc:
```

Comment:
```suggestion
meta_desc: Building an RSS Service with Pulumi
```

Reply:
Ah yes -- I'll add something for this. Thanks for the nudge!